# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/dir.h.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- /dev/null	2005-06-15 01:10:11.182193024 +0200
+++ ./include/sys/dir.h	2005-06-16 01:39:58.000000000 +0200
@@ -0,0 +1,12 @@
+
+#ifndef _SYS_DIR_H
+#define _SYS_DIR_H
+
+#warning "This is a non-standard header - use <dirent.h> instead!"
+
+#include <dirent.h>
+
+#define direct dirent
+
+#endif	/* _SYS_DIR_H */
+
